share/spice/zomato/zomato.js

```diff
@@ -0,0 +1,80 @@
+(function (env) {
+    "use strict";
+    var query_orig = DDG.get_query();
+    //$.getJSON('https://www.zomato.com/duckduckgo/ddg_track?value='+encodeURIComponent(query_orig));
+    env.ddg_spice_zomato = function(api_result){
+        if (!api_result || api_result.error) {
+            return Spice.failed('zomato');
+        }
+        //console.log(api_result);
+        Spice.add({
```

Comment:
@dheeraj143 In order to get maps working, you need to make sure the Maps JS is loaded. Please wrap the `Spice.add()` call in a ` DDG.require('maps', function() { ... }` call. This will fix the `L is undefined` error you're seeing.

The Parking Spice is a great example